code/client/munkiimport

```diff
@@ -800,6 +819,10 @@ def main():
         configure()
         exit(0)
 
+    if options.init:
+        init_repo(arguments[0].strip())
```

Comment:
Fair enough. :) I just spent 30 minutes trying to figure out why munki wouldn't work when I simply dragged the folder to the Terminal window. I looked at the options and none of them seemed like they *should* have leading or trailing whitespace. Maybe just an rstrip() would be better.